server/store/apps.go

```diff
@@ -116,34 +118,62 @@ func (s *appStore) Get(appID apps.AppID) (*apps.App, error) {
 	return nil, utils.NewNotFoundError("app %s is not installed", appID)
 }
 
-func (s *appStore) AsMap() map[apps.AppID]apps.App {
+func (s *appStore) List() []apps.App {
+	var out []apps.App
+	for _, app := range s.ListAsMap() {
+		out = append(out, app)
+	}
+
+	return out
+}
+
+func (s *appStore) ListAsMap() map[apps.AppID]apps.App {
```

Comment:
What was wrong with, `AsMap`, that it was not "raw"? 0/5 perhaps, `ListEnabledApps`, and `MapEnabledApps` (and `RawList`, see below?)